fdb-record-layer-lucene/src/test/java/com/apple/foundationdb/record/lucene/directory/FDBIndexOutputTest.java

```diff
@@ -93,11 +102,7 @@ public void testCopyBytesReadAheadPipelineShortBuffer() throws Exception {
         IndexInput blocks = directory.openChecksumInput(FILE_NAME, IOContext.READONCE);
         output = new FDBIndexOutput(FILE_NAME_TWO, directory);
         output.setExpectedBytes((PrefetchableBufferedChecksumIndexInput) blocks, 2 * FDBDirectory.DEFAULT_BLOCK_SIZE);
-        assertEquals("[(1,0), (1,1)]", directoryCacheToString());
-    }
-
-    private String directoryCacheToString() {
-        return directory.getBlockCache().asMap().keySet().stream().sorted().collect(Collectors.toList()).toString();
+        assertEquals(List.of(Pair.of(1L, 0), Pair.of(1L, 1)), directoryCacheKeys());
```

Comment:
Replaced the string-based assert with object comparison. The `Pair` does have a `toString` method that we could switch to, but this seemed like a more sensible assert anyway.